### ISSUE TITLE
Error while loading on dedicated server (scenario crash)

### ISSUE THREAD

**Issue by Rallo00** [jetelain/Arma3Map#103]
Hi, when trying to start a custom scenario (after lobby) on your map it returns the following error causing the crash of the scenario therefore making Arma 3 load another scenario in lobby. CUP Terrains Core is loaded on the dedicated server.

![error](https://github.com/user-attachments/assets/b5dc970c-e09d-4862-9de0-6466d84fcf05)

Steps to reproduce the error:
- Open Dedicated server
- Load custom scenario created on Sahatra
- Select a role in the lobby
- Start

**Comment by Rallo00** [comment]
Looks like there is a CfgWorld in the config.bin and this error only appears on dedicated server.

**Comment by jetelain** [comment]
Hi, I'm not the author of the map. Can you join the rpt file from the server ? It seems some mods are missing on the server.

**Comment by Rallo00** [comment]
Hi jetelain, unfortunately I am very sure no mods are missing as I developed the scenario. I also created a completely empty scenario on Sahara therefore requiring only CUP Terrains but had the same problem.
I managed to make it work by adding the following:

server.cfg
```
class AdvancedOptions
{
LogObjectNotFound = true; // logging enabled
SkipDescriptionParsing = false; // parse description.ext
ignoreMissionLoadErrors = true;
};
```

With the ignoreMissionLoadErrors the server automatically ignores all errors like the one I posted and keeps loading the scenario. Cannot provide rpt anymore because server privileges have been changed and I am not in charge of the virtual machine anymore.
